network-api/networkapi/management/commands/review_app_admin.py

```diff
@@ -25,4 +29,47 @@ def handle(self, *args, **options):
                 lower_case=True
             ).generate({})
             User.objects.create_superuser('admin', 'admin@example.com', password)
-            self.stdout.write(f'superuser created. Password: {password}')
+
+            reviewapp_name = settings.HEROKU_APP_NAME
+            m = re.search(r'\d+', reviewapp_name)
+            pr_number = m.group()
+
+            # Get PR's title from Github
+            r = requests.get(f"https://api.github.com/repos/mozilla/foundation.mozilla.org/pulls/{pr_number}")
+            pr_title = r.json()['title']
+
+            slack_payload = {
+                "attachments": [
+                    {
+                        "fallback": "New review app deployed :rocket:\n"
+                                    f"PR {pr_number}: {pr_title}\n"
+                                    f"Login: admin\n"
+                                    f"Password: {password}\n"
+                                    f"URL: https://{reviewapp_name}.herokuapp.com",
+                        "pretext":  "New review app deployed :rocket:",
+                        "title":    f"PR {pr_number}: {pr_title}\n",
+                        "text":     "Login: admin\n"
+                                    f"Password: {password}\n",
+                        "color":    "#7CD197",
+                        "actions": [
+                            {
+                                "type": "button",
+                                "text": "Open review app",
+                                "url": f"https://{reviewapp_name}.herokuapp.com"
+                            },
+                            {
+                                "type": "button",
+                                "text": "Open PR on Github",
```

Comment:
Instead of "Open" can we say "View"? When I first saw the text I wondered why I'd need to open a new PR.